components/Heading/Heading.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Heading from './Heading';
+
+describe('Heading component', () => {
```

Comment:
We'll want a separate test for each heading size.

Reply:
Updated with separate test cases for each `h1 - h4`